sdk/containerregistry/Azure.ResourceManager.ContainerRegistry/src/Azure.ResourceManager.ContainerRegistry.csproj

```diff
@@ -1,8 +1,6 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <Version>1.1.0-beta.2</Version>
```

Comment:
Change the version to a stable one.

Reply:
The version will be updated in this prepare release [PR](https://github.com/Azure/azure-sdk-for-net/pull/31090)